nemo/collections/tts/models/speechllm/megatron_t5_speechllm_model.py

```diff
@@ -1504,6 +1646,10 @@ def predict_step(self, batch: Any, batch_idx: int, dataloader_idx: int = 0, log_
                     )
                     output_logits = output_tensor[0]['output_logits']
                     token_and_speech_logits = output_tensor[0]['token_and_speech_logits']
+                    attention_probs = token_and_speech_logits[2]
```

Comment:
@paarthneekhara, `attention_probs=None` if `self.return_all_crossattention_probs=False`. By default it is False. Should we add a `if...else..` here?

Reply:
During inference, IIUC, those `attention_probs` are only used to log the attention map. Maybe we should move these to the places before logging attentin map?